VAMobile/src/utils/appointments.tsx

```diff
@@ -280,35 +297,76 @@ const getListItemsForAppointments = (
   return listItems
 }
 
-const getAppointmentStatus = (
-  isPendingAppointment: boolean,
+const getStatus = (
+  isPending: boolean,
   status: AppointmentStatus,
   t: TFunction,
-  condensedMarginBetween: number,
+  mb: number,
 ): TextLineWithIconProps | undefined => {
   if (status === AppointmentStatusConstants.CANCELLED) {
     return {
       text: t('appointments.canceled'),
       textTag: { labelType: LabelTagTypeConstants.tagInactive },
-      mb: condensedMarginBetween,
+      mb,
     }
   } else if (status === AppointmentStatusConstants.BOOKED) {
     return {
       text: t('appointments.confirmed'),
       textTag: { labelType: LabelTagTypeConstants.tagGreen },
-      mb: condensedMarginBetween,
+      mb,
     }
-  } else if (isPendingAppointment) {
+  } else if (isPending) {
     return {
       text: t('appointments.pending'),
       textTag: { labelType: LabelTagTypeConstants.tagYellow },
-      mb: condensedMarginBetween,
+      mb,
     }
   } else {
     return undefined
   }
 }
 
+const getDate = (startDateUtc: string, timeZone: AppointmentTimeZone): TextLineWithIconProps => ({
+  text: getFormattedDateWithWeekdayForTimeZone(startDateUtc, timeZone),
+  variant: 'MobileBodyBold',
+})
+
+const getTime = (startDateUtc: string, timeZone: AppointmentTimeZone, mb: number): TextLineWithIconProps => ({
+  text: getFormattedTimeForTimeZone(startDateUtc, timeZone),
+  variant: 'MobileBodyBold',
+  mb,
+})
+
+const getCareText = (typeOfCare: string, serviceCategoryName: string, isCovidVaccine: boolean, t: TFunction) => {
+  if (serviceCategoryName === 'COMPENSATION & PENSION') {
+    return t('appointments.claimExam')
+  } else if (isCovidVaccine) {
+    return t('upcomingAppointments.covidVaccine')
```

Comment:
covid vaccines again are no longer a separate type

Reply:
Removed the isCovidVaccine case here